From f007773e819b3206468d1c986fecbf8d542014cb Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Mon, 22 Aug 2016 22:32:39 -0700
Subject: [PATCH] rb-application: Ensure paramater nullability is known to
 bindings

Currently the bindings are led to believe this parameter is not allowed to
recieve null values, which will break Pythonic extensions using the
rb_application_add_accelerator function via GObject Introspection

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 shell/rb-application.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shell/rb-application.c b/shell/rb-application.c
index fe4b3d2..377f06d 100644
--- a/shell/rb-application.c
+++ b/shell/rb-application.c
@@ -914,7 +914,7 @@ rb_application_set_menu_accelerators (RBApplication *app, GMenuModel *menu, gboo
  * @app: the #RBApplication
  * @accel: accelerator string
  * @action: the name of the action to activate
- * @parameter: parameter to pass when activating the action, or NULL if
+ * @parameter: (nullable): parameter to pass when activating the action, or NULL if
  *   the action does not accept an activation parameter.
  *
  * Like #gtk_application_add_accelerator, except the accelerator only applies
-- 
2.9.2

